### DISCUSSION TITLE
Query Sqlite Database or ODBC data source - get SQL statement

### DISCUSSION THREAD

**Discussion by rajendradewani** [microsoft/semantic-kernel#1042] [General]
How can I achieve functionality similar to Langchain SQLite?

I want to connect my SQLite DB and generate SQL statements by asking questions to AI.

Example:
https://python.langchain.com/en/latest/modules/chains/examples/sqlite.html

https://github.com/brunogarcia/langchain-titanic-sqlite

I would appreciate any help.

**Comment by evchaki** [comment]
@rajendradewani  - check this out for SQLite - https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.Sqlite  then you will need to write a function to create SQL - check out this link in our Discord (https://aka.ms/sk/discord) - https://discord.com/channels/1063152441819942922/1098714016890769408/1106207806853501008
